src/app/compliance/actions.js

```diff
@@ -199,3 +200,42 @@ export const removeInvestors = (addresses: Array<Address>) => async (dispatch: F
     true // TODO @bshevchenko
   ))
 }
+
+export const getFreezeStatus = () => async (dispatch: Function, getState: GetState) =>{
+  // $FlowFixMe
+  const transferManager = await getState().token.token.contract.getTransferManager()
```

Comment:
you shouldn't request TransferManager second time, it was already requested in fetchWhitelist ac above, grab it from the state